src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubAuthenticationTests.kt

```diff
@@ -125,20 +128,26 @@ class GithubAuthenticationTests : CustomConfigTests()
 
     private fun assertAuthSuccess(result: Response)
     {
-        assertSuccessful(result)
+        result.use {
```

Comment:
Was getting some warnings about unclosed reponses. I'm not sure this was actually the culprit but can't hurt.